jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/openshift/OpenshiftBuildServiceIntegrationTest.java

```diff
@@ -70,12 +69,11 @@
 
 public class OpenshiftBuildServiceIntegrationTest {
 
-    private static final Logger LOG = LoggerFactory.getLogger(OpenshiftBuildServiceIntegrationTest.class);
-
     private static final int MAX_TIMEOUT_RETRIES = 5;
 
-    @Rule
-    public final OpenShiftServer mockServer = new OpenShiftServer(false);
+    private OpenShiftMockServer mockServer;
+
+    private NamespacedOpenShiftClient openShiftClient;
```

Comment:
Why did we remove the @Rule ?

Reply:
I was facing a strange intermittent issue with OpenShiftBuildServiceIT where it was getting stuck during execution. Here are logs of CI run where I still had OpenShiftServer `@Rule`

https://app.circleci.com/pipelines/github/eclipse/jkube/2212/workflows/2d9d3db0-f0c3-41e5-bb58-1f77ed4cb155/jobs/11469